terminal/src/lib.rs

```diff
@@ -273,6 +276,18 @@ pub mod components {
         println!("{}", style(text).dim());
     }
 
+    pub fn markdown(content: &str) {
+        if !content.is_empty() {
+            blank();
+            if command::bat(["-p", "-l", "md"], content).is_err() {
+                blob(content);
+
+                blank();
+                warning("Could not apply syntax highlighting. Please make sure `bat` is available.")
```

Comment:
I think we decided to remove the warning?

Reply:
Dang, that should not be there anymore. Thanks